toolz/curried.py

```diff
@@ -25,30 +25,41 @@
 """
 
 import toolz
-import toolz.curried_exceptions
-from .functoolz import curry
 import inspect
 
+_numargs = {
+    # built-ins
+    toolz.filter: 2,
+    toolz.map: 2,
```

Comment:
I believe so.  These specify the minimum number of arguments required.  I can update the name of the variable to make that clear.  I should also review the entire PR again, because it's been a while since I've looked at it.